src/chapters/spring/studio.rst

```diff
@@ -0,0 +1,70 @@
+Studio: Skills Tracker
+======================
+
+Wouldn't it be nice to have a small tracker to show us what skills we have built and where we are at in learning them?
+Let's build something that would let us do just that!
+
+As always, read through the whole studio before starting!
+
+At the end of the studio, your final project should be able to:
+
+#. Take input from a user via a form and post the information in a way that is easy to read.
```

Comment:
maybe just scrunch these into a sentence since theres only one item in the list

Reply:
Done!